packages/v4/src/content/design-guidelines/components/label/label.md

```diff
@@ -3,60 +3,75 @@ id: Label
 section: components
 ---
 
-A **label** highlights an element on a page to draw attention to it or make it more searchable. Labels can also be used to tag items of the same category.
-
-If you want to show a count, use a [badge](https://www.patternfly.org/v4/components/badge/react) instead.
+Use a **label** when you want to highlight an element on a page to draw attention to it or make it more searchable. Labels can also be used to tag items of the same category.
 
 ## Usage
-Labels are easily customized to fit different use cases. There are two main types of labels:
-1. Unfilled
+There are 2 label types:
 
-<img src="./img/unfilled-labels.png" alt="Unfilled labels" width="646"/>
+<img src="./img/label-features-type.png" alt="Different label types" width="180"/>
 
-2. Filled
+1. **Default labels:** Use by default when needing a label.
+2. **Compact labels:** Use when confronted by space limitations and a smaller label is required (for example, in a table).
 
-<img src="./img/filled-labels.png" alt="Filled labels" width="646"/>
 
-Use the filled option to add more visual prominence to a label. If your use case requires a mix of clickable and non-clickable labels, consider using one type for clickable and another for non-clickable. Whatever convention you choose, be sure to maintain consistency through your UI. 
 
-The default colors for labels are gray, blue, green, orange, red, purple, and cyan. We offer a wide array of colors to allow for color coding labels within UIs. If desired, label colors can be used to indicate status. Regardless, we recommend you avoid using the red or green labels unless they indicate danger/error or success, respectively.
+Both default and compact labels come in the following default colors: gray, blue, green, orange, red, purple, and cyan. We offer a wide array of colors to allow for color coding labels within UIs. Label colors can be used to indicate status if desired, but do not have to. Regardless, we recommend you avoid using the red label unless it indicates danger or an error state.
```

Comment:
Could you add an image here with the different label colors?
This one that's in the design doc assets doc:
<img width="737" alt="image" src="https://user-images.githubusercontent.com/51165119/160001920-93bf3d64-32a9-476d-a3d6-57ed0682ea39.png">

Reply:
Change has been applied - I reused the filled label image